docs/architecture.md

```diff
@@ -19,7 +19,7 @@ Contour is a client of the Kubernetes API. Contour watches Ingress, Service, and
 The transfer of information from Kubernetes to Contour is by watching the API with the SharedInformer framework.
 The transfer of information from Contour to Envoy is by polling from the Envoy side.
 
-Kubernetes Readiness Probes are configured to check the status of Envoy.
+Kubernetes Liveness and Readiness Probes are configured to check the status of Envoy.
```

Comment:
i don't think any of these are proper nouns. Habitually I capitalise things like Service and Ingress to suggest they are the names of kubernetes objects, not the generic term, but this practice probably doesn't carry water beyond that.